src/Campaigns/resources/admin/components/CampaignsListTable/index.tsx

```diff
@@ -137,8 +152,12 @@ export default function CampaignsListTable() {
                 rowActions={CampaignsRowActions}
                 listTableBlankSlate={ListTableBlankSlate()}
             >
-                <CreateCampaignModal isOpen={isOpen} setOpen={setOpen} />
-                <MergeCampaignModal />
+                <CreateCampaignModal isOpen={isCreateCampaignModalOpen} setOpen={setCreateCampaignModalOpen} />
+                <MergeCampaignModal
+                    isOpen={isMergeCampaignsModalOpen}
+                    setOpen={setMergeCampaignsModalOpen}
+                    campaigns={campaignsToMerge}
+                />
```

Comment:
See my other comment, this will be directly returned using the confirm method so no need to render here.